GiniVision/Classes/GiniPDFDocument.swift

```diff
@@ -56,6 +57,42 @@ final public class GiniPDFDocument: NSObject, GiniVisionDocument {
         }
     }
     
+    fileprivate func renderFirstPage(fromPdf pdf:CGPDFDocument) -> UIImage? {
+        var pdfImage:UIImage?
+        let pdfDoc = pdf
+        
+        if let pdfPage:CGPDFPage = pdfDoc.page(at: 1) {
+            var pageRect:CGRect = pdfPage.getBoxRect(.cropBox)
+            
+            // In case that the image was rotated 90 or 270, final rect should be rotated to portrait             
+            if pdfPage.rotationAngle == 90 || pdfPage.rotationAngle == 270 {
```

Comment:
I would extract that in a method like `normalizeOrientation` or something...

Reply:
I like the idea! 👌